### ISSUE TITLE
No preview of text files when using ranger via tmux

### ISSUE THREAD

**Issue by istudyatuni** [ranger/ranger#2397]
#### Runtime Environment

- Operating system and version: 5.10.49-1-MANJARO
- Terminal emulator and version: gnome-terminal, Version 3.40.3 for GNOME 40
- Python version: 
- Ranger version/commit: 
- Locale: 

```
$ ranger --version
ranger version: ranger-master 
Python version: 3.9.6 (default, Jun 30 2021, 10:22:16) [GCC 11.1.0]
Locale: ru_RU.UTF-8
```

#### Current Behavior

When I try to use ranger inside tmux session, I cant see text files preview: 
![image](https://user-images.githubusercontent.com/43654815/126161792-7d8593b1-7706-4823-adb9-fe09209adeeb.png)

#### Expected Behavior

Previews shown: 
![image](https://user-images.githubusercontent.com/43654815/126161938-17ce8db7-faf7-45ee-abad-7ef1de108060.png)

#### Context

I am working via ssh on another PC, and I don't have good gui file manager and text editor, and ranger is a good tool for viewing what's in a file without having to open it up in an editor speccially.

#### Possible Solutions

I don't know, why this could have happened :)

#### Steps to reproduce
1. Open tmux
2. Open ranger
3. Move cursor to any text file

#### Traceback

```

```

**Comment by toonn** [comment]
If you look closely you can see there is actually text displayed. It's just using horrible formatting. So you probably have bat or highlight or pygments installed.
Do you use the base16 theme perhaps?

**Comment by istudyatuni** [comment]
> So you probably have bat or highlight or pygments installed.

Yes, all of this.

> Do you use the base16 theme perhaps?

I don't know how to check it. But after some experiments (with different terminal emulators) and searching I found this https://stackoverflow.com/q/48430746, and found, that `echo $TERM` without tmux prints `xterm-256color`, but inside tmux prints `screen`, and when I manually set `TERM=xterm-256color`, previews works. Also, here is a link https://unix.stackexchange.com/q/240146 about `TERM` env inside tmux
